cli/main.rs

```diff
@@ -668,7 +668,7 @@ async fn test_command(
   let mut maybe_coverage_collector = if coverage {
     // Ugliness to let the socket serer thread spin up
     (&mut *worker).await?;
-    tokio::time::delay_for(std::time::Duration::from_millis(500)).await;
+    tokio::time::delay_for(std::time::Duration::from_millis(1000)).await;
```

Comment:
Trying to figure out if this is just a race condition, we'll just retry the connection if it is 😅